packages/mds-jurisdiction-service/server/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import logger from '@mds-core/mds-logger'
+import { hours } from '@mds-core/mds-utils'
+import { JurisdictionServiceProvider } from '../service/provider'
+
+const signals: NodeJS.Signals[] = ['SIGTERM', 'SIGINT']
```

Comment:
I'm not following what this `signals` stuff is intended for, could you elaborate? Thanks!

Reply:
If you ctrl-c when running locally or kill the k8s pod, this code will listen for the signal and perform a graceful shutdown of the db connections. This is from the logs when a pod is terminated:
```
│ INFO "Initializing Repostory: jurisdictions"                                                                         │
│ INFO "Initializing ro connection: jurisdictions-ro-353d2a18-c14b-4b6b-ba02-f52896e7048c"                             │
│ INFO "Initializing rw connection: jurisdictions-rw-353d2a18-c14b-4b6b-ba02-f52896e7048c"                             │
│ INFO "Ran no migrations (jurisdictions-migrations)"                                                                  │
│ INFO "Running @container-images/mds-jurisdiction-service v0.0.1 (ce943d73) listening for SIGTERM, SIGINT"            │
│ INFO "Shutting down @container-images/mds-jurisdiction-service v0.0.1 (ce943d73) on SIGTERM"                         │
│ INFO "Terminating Repository: jurisdictions"                                                                         │
│ INFO "Terminating ro connection: jurisdictions-ro-353d2a18-c14b-4b6b-ba02-f52896e7048c"                              │
│ INFO "Terminating rw connection: jurisdictions-rw-353d2a18-c14b-4b6b-ba02-f52896e7048c"
```

Reply:
Monitoring for process termination to perform graceful shutdown.